native/packages/react-native-bpk-component-animate-height/src/BpkAnimateHeight-test.common.js

```diff
@@ -23,6 +23,7 @@ import BpkText from 'react-native-bpk-component-text';
 import BpkAnimateHeight from './BpkAnimateHeight';
 
 const commonTests = () => {
+  jest.useFakeTimers();
```

Comment:
It pains me to say this, but I think each usage of `jest.useFakeTimers();` warrants a comment...

Reply:
this makes me happy, really happy 😄 
will gladly add a comment there